vendor/engines/split_accounts/spec/engine_helper.rb

```diff
@@ -0,0 +1,9 @@
+# Rspec configurations for this engine
+RSpec.configure do |config|
+  
+  # Skip split_accounts tests if split_accounts feature is disabled
+  unless SettingsHelper.feature_on?(:split_accounts)
```

Comment:
I started with the feature_setting metadata but the `@@class` variables and extensions on engine load didn't seem to reset between tests.